B9PartSwitch/PartSwitch/PartSwitchFlightDialog.cs

```diff
@@ -104,7 +106,28 @@ private static DialogGUIBase[] CreateOptions(ModuleB9PartSwitch module, IList<Ca
 
             options.Add(new DialogGUIButton(Localization.PartSwitchFlightDialog_CancelString, delegate { } ));
 
-            return options.ToArray();
+            const float buttonHeight = 35;
+            if (buttonHeight * options.Count < 0.75f * Screen.height) return options.ToArray();
+
+            // UnityEngine.TextRenderingModule must be added to project References
```

Comment:
Do we need this comment.  If the dependency is documented in the commit/PR it should be findable enough

Reply:
I just wanted to make sure I didn't forget it.  Comment removed.